src/services/students/ostap-blystsiv.ts

```diff
@@ -90,33 +126,39 @@ export class OstapBlystsivConfigLoaderService implements IConfigLoaderService {
       email: this.getStudentEmail(),
       projectName: 'Rocket',
       personalProjectLink: 'https://google.com',
-      header: Constants.DefaultConfig.header,
+      header: generateHeaderFrame(),
       frames: generateFrames(),
     };
   }
 }
 
-function generateFrames(): IFrame[] {
+const generateFrames = (): IFrame[] => {
   const frames: IFrame[] = [];
 
-  for (let frameNumber = 0; frameNumber < 30; frameNumber++) {
+  if (FRAMES_COUNT < 30) {
```

Comment:
What is the point of this check ?
It is never going to be executed so we might as well remove it.